### ISSUE TITLE
Error connecting to database.

### ISSUE THREAD

**Issue by covert1** [prashants/webzash-v1-defunct#7]
error message, holding up progress.

**Comment by ldphoto** [comment]
Hi!  I'm new to this forum so hopefully people will be able to read this.  I'm trying to figure out how to connect to a database without creating one in the system.  I have been able to get my one account working for FY2013 but I tried to create a carry forward account from 2012 to carry over my assets into the business and see what happens.  I am not sure if I need a seperate database for this account or if it should connect to my current database.  

A little background about myself, is that I am a website designer and unfortunately not a very good developer.  I can read most of PHP/MySQL language and copy/paste some things to make them work but I'm really slow with it and am pretty much in the dark if I had to program things on my own.  So Most likely I'm a lot less savvy then most of you all but I can get around pretty well.  I also have a Business degree so I'm familiar with Accounting debits and credits but I just need to brush off the old accounting books to remember what everything is really supposed to do!

I hope you all can help and I'll be checking the forums for any responses I may get! 

Donny P
